src/test/scala/ExampleWorkflowSpec.scala

```diff
@@ -0,0 +1,87 @@
+import org.apache.commons.io.FileUtils
+import org.pentaho.di.trans.step.StepMetaInterface
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import uk.gov.nationalarchives.pdi.step.jena.model.JenaModelStepMeta
+import uk.gov.nationalarchives.pdi.step.jena.serializer.JenaSerializerStepMeta
+import uk.gov.nationalarchives.pdi.step.jena.shacl.JenaShaclStepMeta
+import uk.gov.nationalarchives.pentaho.{ DatabaseManager, QueryManager, WorkflowManager }
+
+import java.io.File
+import scala.reflect.io.Directory
+import scala.util.{ Failure, Success }
+
+class ExampleWorkflowSpec extends AnyWordSpec with Matchers with BeforeAndAfterAll {
+
+  private val exampleWorkflow = "example.ktr"
+  private val outputDirectory = "output"
+  private val resultFilename = "example.ttl"
+  private val shaclDirectory =
+    "/home/rkw/Source/GitHub/nationalarchives/tna-cat/Omega/ildb/export/kettle/resources/shacl"
+  private val shaclFilename = "odrl-shacl.ttl"
+  //private val dbDriverClass = "org.h2.Driver"
+  private val databaseDir = "./data-dir"
```

Comment:
I am a bit worried that this is hard-coded to a sub-directory of the project. It should likely be a temporary folder per test suite that is removed after use.